docs/cloud-config.md

```diff
@@ -16,6 +16,40 @@ subnet-id=
 security-group-id=
 ```
 
+### Examples for HuaweiCloud
+
+```
+[Global]
+access-key=******
+secret-key=******
+project-id=******
+region=ap-southeast-1
+cloud=myhuaweicloud.com
+auth-url=https://iam.myhuaweicloud.com:443/v3
+
+[Vpc]
+id=1jk3u4ic4******02361dde2
+subnet-id=1183fe******bfdfc10d
+security-group-id=1c308fe5******be519e76f02
+```
+
+### Examples for Orange Cloud
```

Comment:
better to use Flexible Engine instead of Orange Cloud

Reply:
done